core/data/src/main/java/me/nalab/core/data/user/UserOAuthInfoEntity.java

```diff
@@ -0,0 +1,41 @@
+package me.nalab.core.data.user;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
+import me.nalab.core.data.common.TimeBaseEntity;
+
+@Table(name = "user_oauth_info)")
+@Entity
+@Getter
+@SuperBuilder
+@AllArgsConstructor
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class UserOAuthInfoEntity extends TimeBaseEntity {
+	@Id
+	private Long id;
+
+	@Column(nullable = false, updatable = false)
+	private String provider;
+
+	@Column(nullable = false)
+	private String email;
+
+	@Column(nullable = true)
+	private String username;
+
+	@Column(nullable = true)
```

Comment:
저희 DB 이름을 아래 처럼 under bar 형식으로 했는데, camel case를 자동으로 변형해주는지?? 모르겠네요

```suggestion
	@Column(nullable = true, name = "phone_number")
```

Reply:
[Naming strategies](https://docs.jboss.org/hibernate/orm/6.2/userguide/html_single/Hibernate_User_Guide.html#naming) 여기보면 명시 안하면 `default(=jpa)`로 적용되고 아래 코드보면 AttributePath가져와서 해당 property 가져오는 데 이후에 따로 변환없이 그대로 쓰더라구요!
```
public class ImplicitNamingStrategyJpaCompliantImpl implements ImplicitNamingStrategy, Serializable {
	@Override
	public Identifier determineBasicColumnName(ImplicitBasicColumnNameSource source) {
		// JPA states we should use the following as default:
		//     "The property or field name"
		// aka:
		//     The unqualified attribute path.
		return toIdentifier( transformAttributePath( source.getAttributePath() ), source.getBuildingContext() );
	}
	
	protected String transformAttributePath(AttributePath attributePath) {
		return attributePath.getProperty();
	}
}
```

Reply:
b215794